Wiki/Modules/FvwmRearrange/index.md

```diff
@@ -27,4 +27,65 @@ current screen subject to certain constraints. Layering is performed
 so consecutive windows will have their window titles visible underneath
 the previous.
 
+## Configuration and Use
 
+FvwmRearrange is invoked from a menu, pop-up, or button. Several
+command line options can be used to constrain the layering. For a full
+list of FvwmRearrange configuration options see the FvwmRearrange manpage.
+
+Here is a sample configuration for tiling with 2 columns.
+
+{% highlight fvwm %}
+FvwmRearrange -tile -r -mn 2 -maximize 0 0
+{% endhighlight %}
+
+To restore the windows, FvwmRearrange doesn't reverse the options,
+use this for all.
+
+{% highlight fvwm %}
+All (CurrentPage, !Iconic, CirculateHit, !Sticky) Maximize Off
+{% endhighlight %}
+
+## Example
+
+This advanced example is attached to key binding (alt-t) with functions that
+do the tiling with 3 columns. With the same key binding, it restores all
+windows. This configure does like the following to the windows, before and
```

Comment:
"This example does like the following to the windows" sounds odd to me.  I would suggest:

"The following example demonstrates this".